packages/app/src/app/constants.ts

```diff
@@ -1,3 +1,10 @@
 export const DIALOG_TRANSITION_DURATION = 0.25;
 export const DIALOG_WIDTH = 420;
 export const REPLY_TRANSITION_DELAY = 0.5;
+export const SUBSCRIPTION_DOCS_URLS = {
```

Comment:
we should move towards a `paths` like structure, similar with v2 and unify it it here with the `urls` from common (getting rid of those dependencies in the process)

Added in note in the RFC: https://www.notion.so/codesandbox-app/Client-codebases-merge-e63037d544814407be282d5a4c94dfd2